tests/framework/common.go

```diff
@@ -489,6 +491,10 @@ func (td *OsmTestData) InstallOSM(instOpts InstallOSMOpts) error {
 		)
 	}
 
+	instOpts.SetOverrides = append(instOpts.SetOverrides,
+		fmt.Sprintf("OpenServiceMesh.certKeyBitSize=%d", instOpts.CertBitSize),
+	)
+
```

Comment:
We shouldn't need this here, we should set a default key size in `pkg/configurator` if not specified via `--set`.